frontend/src/App.js

```diff
@@ -1,66 +1,64 @@
+/*
+ANIME BENSALEM, BRIAHANA MAUGÉ, JOSEPH P. PASAOA
+APP MAIN | Capstone App (Pursuit Volunteer Mgr)
+*/
+
+
+/* IMPORTS */
 import React, { useState, useEffect } from 'react';
 import { Switch, Route, useHistory } from 'react-router-dom';
 import axios from 'axios';
 
 import './App.scss';
+import LoginSignupGate from './Components/LoginSignupGate';
 import LoginSignup from './Components/LoginSignup/LoginSignup';
-import Feedback from './Components/Feedback';
-import VolunteerSearch from './Components/VolunteerSearch';
+import PrivateRouteGate from './Components/PrivateRouteGate';
+import NavBar from './Components/NavBar';
+import Dashboard from './Components/Dashboard';
 import ProfilePage from './Components/Profile/ProfilePage';
 import AdminTools from './Components/AdminTools/AdminTools';
+import VolunteerSearch from './Components/VolunteerSearch';
+import Feedback from './Components/Feedback';
 
 
-function App() {
-  const history = useHistory();
-
+const App = () => {
   const [ loggedUser, setLoggedUser ] = useState({});
+  const [ currentRole, setCurrentRole ] = useState("");
+  const [ isUserStateReady, setIsUserStateReady ] = useState(false);
   const [ feedback, setFeedback ] = useState(null);
 
-  const [ formType, setFormType ] = useState('login');
```

Comment:
To be honest I rather having all these state variables at App lever (the way it was) because they are used in more then one component and is we take them from here, we have to create them for each of the other components

Reply:
no prob. i moved them back into App.js